op-supervisor/supervisor/backend/backend.go

```diff
@@ -36,13 +35,22 @@ type SupervisorBackend struct {
 	// Write = set of chains is changing.
 	mu sync.RWMutex
 
+	// depSet is the dependency set that the backend uses to know about the chains it is indexing
 	depSet depset.DependencySet
 
-	// db holds on to the DB indices for each chain
-	db *db.ChainsDB
+	// chainDBs holds on to the DB indices for each chain
+	chainDBs *db.ChainsDB
```

Comment:
just changing the name of this to match the other "chainXYZ" structures this holds